epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -379,5 +399,25 @@ <h6>(vi)</h6>
 				</section>
 			</section>
 		</section>
+		<section id="exemptions">
+			<h2>Metadata to describe possible exemptions</h2>
+			
+			<p>The European Accessibility Act foresee three possible exemptions to meeting its requirements:</p>
+			
+			<ul>
+				<li>to microenterprises providing services that employ fewer than ten people and have an annual turnover not exceeding 
+					EUR 2 million or an annual balance sheet total not exceeding EUR 2 million;</li>
+				<li>when compliance with accessibility requirements would impose a disproportionate burden on economic operators as 
+					defined in Annex VI of the Directive;</li>
+				<li>when the compliance with accessibility requirements require a significant change in a product or service 
+					that results in the fundamental alteration of its basic nature as described in article 4 of the Directive.</li>
+			</ul>
+
+			<p>Metadata standards for digital publishing ONIX (via <a href="https://ns.editeur.org/onix/en/196">Code List 196</a>)
+				and EPUB metadata (via <a href="https://www.w3.org/TR/epub-a11y-exemption/">The EPUB Accessibility exemption property</a>)
+				provide specific codes for communicating to the digital publishing supply chain (and the end user) any exceptions related 
+				to specific publications. In any case economic operators will be responsible to provide the necessary documentation to 
+				support the exception they invoke.</p>
```

Comment:
I do not know about the politics here, but from a usual W3C style perspective it may be better to put here a formal reference to the exemption note, so that it would also appear in the references' section below.

Reply:
em dashes might also help the sentence here, as the references to onix and epub don't flow with the rest: "Metadata standards for digital publishing &#8212; ... &#8212; provide specific codes..."

The specific standards might also be better written into a note after the paragraph. For example: "NOTE: Refer to ONIX Code List 196 [onix] and The EPUB Accessibility exemption property [epub-a11y-exemption] for more information on expressing these codes."